security/registries.adoc

```diff
@@ -0,0 +1,92 @@
+[[security-registries]]
+= Registries
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-registries-where-do-your-containers-come-from]]
+== Where Do Your Containers Come From?
+
+As discussed, there are tools you can use to scan and track the contents of your
+downloaded and deployed container images. However, there are many public sources
+of container images. When using public container registries, you can add a layer
+of protection by using trusted sources.
+
+[[security-registries-immutable-and-certified-containers]]
+== Immutable and Certified Containers
+
+Consuming security updates is particularly important when managing _immutable
+containers_. Immutable containers are containers that will never be changed
+while running.
+
+Container images are a nicely packaged, layered collection of binaries. When a
+bug is discovered in a deployed binary, you do not edit the binary in place; you
+fix the bug in the source code, rebuild the binary, and replace the broken
+binary with the newly rebuilt binary.
```

Comment:
There's a few "you do not X"'s here. Are these supposed to be recommendations? Like "don't do X". I'm not actually sure what they're saying...